packages/next-app/src/wagmi/client.ts

```diff
@@ -9,9 +11,8 @@ import {
   polygonZkEvm as polygonZkEvmChain,
   sepolia,
 } from "wagmi/chains";
-import { publicProvider } from "wagmi/providers/public";
 
-import { configureChains, createClient } from "#/wagmi";
+const projectId = "4f98524b2b9b5a80d14a519a8dcbecc2";
```

Comment:
Where's exactly this coming from? Again, something that must absolutely be explaining in the code.